Library/Homebrew/rubocops/checksum_cop.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "./extend/formula_cop"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class Checksum < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          %w[Stable Devel HEAD].each do |name|
+            next unless spec_node = find_block(body_node, name.downcase.to_sym)
+            _, _, spec_body = *spec_node
+            audit_checksums(spec_body, name)
+            resource_blocks = find_all_blocks(spec_body, :resource)
```

Comment:
I believe that the resource blocks are currently being audited as part of running `audit_specs`, but the code structure relies on the ResourceAuditor class, and the `audit_checksum` it contains. I think that gap might be part of the cause of this issue.